src/uri/Uri1002.kt

```diff
@@ -0,0 +1,18 @@
+package uri
+
+import java.util.*
+
+fun main() {
+    var raio = Scanner(System.`in`).nextFloat()
+
+    uri1002(raio)
+}
+
+fun uri1002(raio: Float) {
+    var pi = 3.14159
+    var area = pi * raio * raio
+
+    println("A=%.4f".format(area))
+
+
```

Comment:
Só enxuguei mais a tua solução.

```suggestion
fun main() {
    val radius = Scanner(System.`in`).nextFloat()
    uri1002(radius)
}

fun uri1002(radius: Float) {
    val area = Math.PI * radius * radius
    println("A=%.4f".format(area))
}
```